### DISCUSSION TITLE
Create New Company Button

### DISCUSSION THREAD

**Discussion by irik77587** [timschofield/webERP#164] [Development Discussion & Specification]
I searched to net to find the easiest way to create company is : to navigate to Z_MakeNewCompany.php. So I made it simple by adding a line to /include/header.php
`
'<li><a href="', $RootPath, '/Z_MakeNewCompany.php">', _('New Company'), '</a></li>',
`
like this:
```

<ul id="menu">
	<li><a href="', $RootPath, '/index.php">', _('Main Menu'), '</a>',
	'<li><a href="', $RootPath, '/Z_MakeNewCompany.php">', _('New Company'), '</a></li>';

```
This adds a new option **New Company** in **QuickMenu**
However, adding this option in MainMenu should be a better option. Because, people don't often make a new company.

**Comment by timschofield** [comment]
Does this script still work? I haven't tried it in many years, and it may throw some errors.

That said, if you want to create an entry into the main menu then you just need to create entries in the includes/MainMenuLinksArray.php script.

Thanks
Tim

**Comment by dalers** [comment]
I recall seeing in the manual how to create a new company, it referenced Z_MakeNewCompany.php.

imho testing the script (and leave the manual as-is) would be the better long-term solution.
